_content/news/_posts/2018-12-02-eindejaarsactie.markdown

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title:  "Eindejaarsactie 2018"
+date:   2018-12-02 20:00:00
+description: Pralines, wijn of ijs ten voordele van Basket Lummen.
+cover: /news/img/leonidas2.jpg
+permalink: /news/2018-12-02-eindejaarsactie/
+---
+
+Zoek je een leuk eindejaarscadeautje voor je familie, je vrienden of jezelf? Dan vind je [hier](/news/downloads/eindejaarsactie2018b.pdf) zeker iets naar jouw smaak! 
+
+Voor deze aanbieding slaan deze partners de handen in elkaar, ten voordele van Basket Lummen:
```

Comment:
> Voor deze aanbieding slaan deze partners de handen in elkaar

2x deze kort na elkaar, misschien veranderen naar

Voor deze aanbieding slaan volgende partners de handen in elkaar

Reply:
ok